src/utils/file.ts

```diff
@@ -19,22 +14,32 @@ export function getStoredHashedValues() {
   } catch (e: any) {
     console.error(`Unable to read file info: ${e.message}`);
     storeHashedValues({});
+    content = fs.readFileSync(filePath, { encoding: 'utf8' });
     try {
       return JSON.parse(content);
     } catch (e: any) {
       console.error(`Unable to read file info: ${e.message}`);
     }
   }
-  return null;
+  return {};
 }
 
-export function getFile(name: string) {
-  let file_path = getSoftwareDir();
-  if (isWindows()) {
-      return `${file_path}\\${name}`;
+export function getFile(name: string, default_data: any = {}) {
```

Comment:
Should default_data always be an object?

Reply:
Will this function ever be called with default_data? I only see it being called once, without default_data.

Reply:
oh, this defaults to an object like you see there so if its not passed in and the file is not found, it will create the file with the default data. Its only being used by one caller, thats true. I can change this to specifically fetch this file until we have a new use case.

Reply:
i've added the default value to the caller and updated the getFile to default to null. If that arg isn't passed in it will not try to create it with a default value